docs/faqs/max_execution_time.rst

```diff
@@ -8,31 +8,58 @@ To ensure fairness, there is a maximum execution time for each Qiskit Runtime jo
 a job exceeds this time limit, it is forcibly cancelled. This is represented in the job
 status as `Cancelled - Ran too long`. The maximum execution time is the
 smaller of 1) the system limit and 2) the ``max_execution_time`` defined by the program.
+
+System limit
+--------------
+
 The system limit is defined as follows:
 
 Qiskit Runtime on IBM Cloud
----------------------------
+***************************
 
 The system limit on the job execution time is 3 hours for a job running on a simulator
 and 8 hours for a job running on a physical system.
 
 Qiskit Runtime on IBM Quantum
------------------------------
+*****************************
 
-The system limit on the job execution time is
+The system limit on the job execution time is described in the following table:
 
 +------------------+--------------+-----------+--------------+-----------+
 |                  | Public Program           | Private Program          |
```

Comment:
Since we no longer talk about public vs private programs, perhaps we should say "Primitives and prototype programs", instead of public program. This would also match the terms used in the Program time limits section.